.github/workflows/go_lint_and_test.yaml

```diff
@@ -0,0 +1,65 @@
+# Reusable workflow for the repositories of the Arcalot organization.
+# This workflow provides linting and testing of Go sources and produces
+# a coverage report.  (It is only run when invoked by another workflow.)
+on:
+  workflow_call:
+env:
+  go_version: 1.21.7
+jobs:
+  golangci-lint:
+    name: golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Golang
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.go_version }}
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v4
+        with:
+          version: v1.55.2
+          args: --timeout=3m
```

Comment:
We run the *linter* with a timeout? Lotta lint, eh? 😉

Reply:
😁 That was in the code that I "borrowed", so I figured I would leave it.  (Somebody must have thought it was a good idea...so who am I to say otherwise!)

Reply:
Huh: I guess I don't have full rights in the root `arcalot` repo: I was able to *make* a comment, but I can't mark it *resolved*. 😦 Which I would otherwise be doing ...